rasa/model.py

```diff
@@ -128,16 +128,15 @@ def should_retrain_nlu(self) -> bool:
         return self.force_training or self.nlu
 
 
-def get_model(model_path: Text = DEFAULT_MODELS_PATH) -> TempDirectoryPath:
-    """Get a model and unpack it. Raises a `ModelNotFound` exception if
-    no model could be found at the provided path.
+def verify_model_path(model_path: Text = DEFAULT_MODELS_PATH) -> Text:
+    """Verifies that a model path exists.
 
     Args:
         model_path: Path to the zipped model. If it's a directory, the latest
                     trained model is returned.
```

Comment:
please also document the `Returns` part (`pydocstyle` unfortunately has a bug which doesn't flag missing `Returns` 😬 )